### ISSUE TITLE
As a user with a disability, I should be able to scroll through the feed using a screen reader.

### ISSUE THREAD

**Issue by dmac1108** [dmac1108/a-flyer-on-the-wall-client#6]
Estimate to complete: 30 minutes
Add appropriate aria elements to feed components
